doc/guide/build.hpp

```diff
@@ -8,6 +8,18 @@ FreeBSD etc). However, mlpack is in the repositories of many Linux distributions
 and so it may be easier to use the package manager for your system.  For example, 
 on Ubuntu, you can install mlpack with the following command:
 
+@code
+$ sudo apt-get install mlpack-bin
+@endcode
+
+On Fedora or Red Hat:
+
+@code
+$ sudo dnf install mlpack
```

Comment:
I'm not sure what this package entails, lib + executables?

Reply:
@zoq  I got this info from here https://www.mlpack.org/doc/stable/doxygen/cli_quickstart.html .But looks like mlpack packages don't exist on fedora. I did dnf list all on a fedora instance and found that there is no such package that matches mlpack string on Fedora repository.If it exists by alternate name please suggest one and I will change it. Else i will remove this line as mlpack may not exist on fedora/red hat repositories

Reply:
I checked on Red Hat 8.3.1-5 dont know if earlier versions support.

Reply:
@rcurtin is the maintainer of the package, so pretty sure he knows.

Reply:
This is just the library.  `mlpack-bin` will contain the command-line programs.

Reply:
But @rcurtin is mlpack their in ;latest red hat repositories. It's not their in Red hat 8.3.1 I believe.